sdk/schemaregistry/azure-data-schemaregistry-avro/src/main/module-info.java

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+module com.azure.data.schemaregistry.serde.avro {
+    requires transitive com.azure.core;
+
+    exports com.azure.data.schemaregistry.serde.avro;
```

Comment:
This package doesn't exist, this should be `com.azure.data.schemaregistry.avro` right?

Reply:
Yeah I missed this, there have been a lot of package changes.